hintman.cabal

```diff
@@ -88,11 +88,14 @@ executable hintman
 test-suite hintman-test
   type:                exitcode-stdio-1.0
   hs-source-dirs:      test
+  other-modules:       Hintman.Suggest
```

Comment:
Let's add `Test.` prefix for test modules to distinguish them from library modules